src/yaws_server.erl

```diff
@@ -3449,20 +3449,21 @@ handle_out_reply(ok, _LineNo, _YawsFile, _UT, _ARG) ->
     ok;
 
 handle_out_reply({'EXIT', Err}, LineNo, YawsFile, _UT, ARG) ->
+    St = ?stack(),
     L = ?F("~n~nERROR erlang  code  crashed:~n "
            "File: ~s:~w~n"
            "Reason: ~p~nReq: ~p~n"
            "Stack: ~p~n",
-           [YawsFile, LineNo, Err, ARG#arg.req, ?stack()]),
-    handle_crash(ARG, L);
+           [YawsFile, LineNo, Err, ARG#arg.req, St]),
+    handle_crash(ARG, L, St);
 
 handle_out_reply({throw, Class, Exc, St}, LineNo, YawsFile, _UT, ARG) ->
     L = ?F("~n~nERROR erlang code threw an uncaught exception:~n "
            "File: ~s:~w~n"
            "Class: ~p~nException: ~p~nReq: ~p~n"
            "Stack: ~p~n",
```

Comment:
It could be argues that including the stack trace in the formatted message is unnecessary, but maybe people are relying on that.